openmc/opencg_compatible.py

```diff
@@ -905,7 +916,7 @@ def get_opencg_lattice(openmc_lattice):
     for z in range(dimension[2]):
         for y in range(dimension[1]):
             for x in range(dimension[0]):
-                universe_id = universes[x][dimension[1]-y-1][z].id
+                universe_id = universes[x][y][z].id
```

Comment:
This is up-to-date with the latest comments.
On Jan 2, 2016 12:14 AM, "sam" <notifications@github.com> wrote:

> In openmc/opencg_compatible.py
> <https://github.com/mit-crpg/openmc/pull/539#discussion_r48682529>:
>
> > @@ -905,7 +916,7 @@ def get_opencg_lattice(openmc_lattice):
> >      for z in range(dimension[2]):
> >          for y in range(dimension[1]):
> >              for x in range(dimension[0]):
> > -                universe_id = universes[x][dimension[1]-y-1][z].id
> > +                universe_id = universes[x][y][z].id
>
> Shouldn't this be universe_id = universes[z][y][x].id?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/539/files#r48682529>.
>